doc/tools/west/index.rst

```diff
@@ -78,11 +216,61 @@ West is:
 - **Clearly specified**: West's behavior in cases where it wraps other
   commands is clearly specified and documented. This enables
   interoperability with third party tools, and means Zephyr developers
-  can always find out what is happening "under the hood" when using West.
+  can always find out what is happening "under the hood" when using west.
+
+See `Zephyr issue #6205`_ and for more details and
+discussion.
+
+.. _no-west:
+
+Using Zephyr without west
+*************************
+
+There are many valid usecases to avoid using a meta-tool such as west which has
+been custom-designed for a particular project and its requirements.
+This section applies to Zephyr users who fall into one or more of these
+categories:
+
+- Already use a multi-repository management system (for example Google repo)
```

Comment:
So there's a tradition for this sort of tool to have un-googlable names...

Reply:
LOL

Reply:
😅 I must admit I had not considered this until you brought it up. I guess you'll need to google "zephyr west". But even then you may get results about the western wind 😮